### DISCUSSION TITLE
user tracking rate limit

### DISCUSSION THREAD

**Discussion by kass507** [plhery/node-twitter-api-v2#215] [Q&A]
I'm thinking of using this package to track a user (in this case a verified person).
My intention is to track every 2 minutes of your location and change of banner and profile picture.

Is there any risk in terms of monitoring limit per minute or maybe every 2 minutes?

I'm about to use this:
`const foundUsers = await client.v1.searchUsers(' ');`

**Comment by alkihis** [comment]
Any risk of what? Being banned from Twitter?

I don't know, this question should more be asked in the official Twitter dev forum? 😄
